README.md

```diff
@@ -140,23 +140,86 @@ sudo openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout /etc/ssl/privat
 
 *Important*: Make sure that the certificate files are named exactly `ssl-certificate.key` and `ssl-certificate.crt`. If you have procured the certificate from a different source, rename them to the prescribed filenames, and ensure that they are located in the same folder as the `install.sh` script. They will get uploaded to the Chef Habitat supervisor during the install.
 
+### Preparing for an Airgapped Installation (Recommended if applicable)
+
+In order to install the on-prem Chef Habitat Builder in an airgapped (no direct Internet access) environment, the following preparatory steps are helpful.
+
+> Note: Unless otherwise noted, the tasks are intended to be completed on a Non-Airgapped environment with Internet connectivity
+
+1. Download the [Zip archive](https://github.com/habitat-sh/on-prem-builder/archive/master.zip) of the on-prem-builder repo
+
+    ```bash
+    curl -LO https://github.com/habitat-sh/on-prem-builder/archive/master.zip
+    ```
+
+1. Download the Chef Habitat [cli tool](https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-%24latest-x86_64-linux.tar.gz?bt_package=hab-x86_64-linux)
+
+    ```bash
+    curl -Lo hab.tar.gz https://api.bintray.com/content/habitat/stable/linux/x86_64/hab-%24latest-x86_64-linux.tar.gz?bt_package=hab-x86_64-linux
+    ```
+
+1. Create the Habitat Builder package bundle from the [Builder Seed List](https://github.com/habitat-sh/on-prem-builder/blob/master/package_seed_lists/builder_x86_64-linux_stable) package seed list and download it
+
+     ```bash
+     git clone https://github.com/habitat-sh/on-prem-builder.git
+     export DOWNLOAD_DIR=/some/base/download/directory
+     cd on-prem-builder
+     hab pkg download --target x86_64-linux --channel stable --file package_seed_lists/builder_x86_64-linux_stable --download-directory ${DOWNLOAD_DIR}/builder_packages
+     ```
+
+1. Create any additional package bundles to upload to Builder from package seed lists as documented in the [Bootstrap Builder](https://github.com/habitat-sh/on-prem-builder/blob/master/README.md#bootstrap-builder-with-habitat-packages-new) section of this README. You can specify `--download-directory ${DOWNLOAD_DIR}/builder_bootstrap` argument to the download command in order to consolidate all bootstrap packages in a single directory
+1. Zip up all the above content, transfer and unzip on the Linux system where Builder will be deployed in the Airgapped environment
+
+> Note: The following tasks are intended to be completed on the Airgapped system where Builder will be deployed, in advance of the installation
```

Comment:
Should this section be part of the "Installation" section below, as this is actually the airgap "install", correct?

Reply:
To me it feels more "preparation" work, whereas the install itself is just intuitively running the `./install.sh` script. If this prep work was done in advance, the `install.sh` is the same unifying experience irregardless.

To help strengthen that stance, I've renamed the "Preparatory Tasks for Airgapped ..." to "Prerequisite Tasks for an Airgapped Installation (Required if applicable)" and put a note in the Installation section that reads:

> Note: If the on-prem Builder system is in an Airgapped (non-Internet connected) environment, you must first complete the prerequisite tasks detailed earlier.

Take a look and see what you think of the flow from a new reader's perspective:

https://github.com/habitat-sh/on-prem-builder/tree/jeremymv2/airgap_and_download_upload#prerequisite-tasks-for-an-airgapped-installation-required-if-applicable 

Is that an improvement that addresses your concern adequately?

Reply:
Cool, that looks good.